terraform/ec2/linux/install-efs-utils.sh

```diff
@@ -0,0 +1,32 @@
+# install amazon EFS client on linux, agnostic to yum, apt, or zypper + rpm
+# https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html
+
+if command -v apt-get >/dev/null; then
+  sudo apt-get update -y
+  sudo apt-get -y install git binutils
+  git clone https://github.com/aws/efs-utils
+  cd ./efs-utils
+  ./build-deb.sh
+  sudo apt-get -y install ./build/amazon-efs-utils*deb
+elif command -v yum >/dev/null; then
+  sudo yum update -y
+  sudo yum install sudo yum install -y amazon-efs-utils
+elif command -v zypper >/dev/null; then
```

Comment:
What os uses zypper? Are we going to add tests for this os?

Reply:
Agree. If adding Zypper, then it would be better to have OpenSUSE or SLES OS too. If not, we are over-engineering this though.

Reply:
We suport SLES according to [public cwa docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html), and `amazon-efs-utils` insallation uses zypper on SLES and OpenSUS according to this [doc](https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html)

Reply:
I think Seth meant to adding a OS use this to confirm this behavior ! In your case, its OpenSus and SLES (if we are not confirming it and breaking at some points, its a tech debt though)

Reply:
Since public docs state that cwa suports SLES, then we will definitely add tests for SLES eventually. 

As for tech debt, I agree that this is script is a tech debt. I plan to migrate the efs mounting process from terraform to the AMI building stage, which should be a much cleaner solution for everyone.